src/utils/bundleReport.js

```diff
@@ -0,0 +1,106 @@
+const path = require('path');
+const sizeTypes = ['B', 'KB', 'MB', 'GB'];
+
+function spaces(amount) {
+  let spaces = '';
+  for (let i = 0; i < amount; i++) {
+    spaces += ' ';
+  }
+  return spaces;
+}
+
+function writeRow(columnWidth, items) {
+  let res = '';
+  if (Array.isArray(items)) {
+    for (let i = 0; i < items.length; i++) {
+      let item = items[i];
+      let itemWidth = Array.isArray(columnWidth) ? columnWidth[i] : columnWidth;
+      item =
+        item.length + 1 > itemWidth
+          ? path.basename(item).substring(0, itemWidth)
+          : item;
+      res += item + spaces(Math.abs(itemWidth - item.length));
+    }
+  }
+  return res;
+}
+
+function prettifySize(size) {
+  let type = 0;
+  while (size > 1024) {
+    size = size / 1024;
+    type++;
+  }
+  return `${size.toFixed(2)} ${sizeTypes[type]}`;
```

Comment:
While unlikely, you could overflow your array in which case you'd end up with `1.54 undefined`. Should at least safe-guard against it and stop dividing by 1024.

Reply:
Ow yeah, thought it wouldn't be an issue as a bundle larger than 1024 GB would be insane for a web application (or any other application)